samples/dynamodb/lambda/DynamoDB_1/Sales/OrderSaga.cs

```diff
@@ -0,0 +1,86 @@
+﻿using System;
+using System.Threading.Tasks;
+using NServiceBus;
+using NServiceBus.Logging;
+using Messages;
+
+namespace SagaSample
+{
+
+  #region OrderSaga
+
+  public class OrderSaga : Saga<OrderSagaData>,
+    IAmStartedByMessages<PlaceOrder>,
+    IHandleMessages<CustomerBilled>,
+    IHandleMessages<InventoryStaged>,
+    IHandleTimeouts<CancelOrder>
+  {
+    static ILog log = LogManager.GetLogger(typeof(OrderSagaData));
+
+    protected override void ConfigureHowToFindSaga(SagaPropertyMapper<OrderSagaData> mapper)
+    {
+      mapper.MapSaga(sagaData => sagaData.OrderId)
+       .ToMessage<PlaceOrder>(s => s.OrderId)
+       .ToMessage<CustomerBilled>(s => s.OrderId)
+       .ToMessage<InventoryStaged>(s => s.OrderId)
+       .ToMessage<CancelOrder>(s => s.OrderId);
+    }
+
+    public async Task Handle(PlaceOrder message, IMessageHandlerContext context)
+    {
+      log.Info($"Placing order: {Data.OrderId}");
+
+      var sendOptions = new SendOptions();
+      sendOptions.RouteToThisEndpoint();
+
+      await context.Publish(new OrderReceived()
+      {
+        OrderId = message.OrderId
+      });
+
+      // Delay 10 seconds to allow time for order cancellation
+      await RequestTimeout<CancelOrder>(context, DateTimeOffset.UtcNow.AddSeconds(15));
```

Comment:
or change the comment

```suggestion
      await RequestTimeout<CancelOrder>(context, DateTimeOffset.UtcNow.AddSeconds(10));
```